types/eleventy/collection-item.d.ts

```diff
@@ -16,6 +16,90 @@
 
 declare global {
   export interface EleventyCollectionItem {
+    /**
+     * The full path to the source input file (including the path to the input directory)
+     */
+    inputPath: string;
+    /**
+     * Mapped from the input file name, useful for permalinks. Read more about [`fileSlug`](https://www.11ty.dev/docs/data-eleventy-supplied/#fileslug).
+     */
+    fileSlug: string;
+    /**
+     * The full path to the output file to be written for this content
+     */
+    outputPath: string;
+    /**
+     * URL used to link to this piece of content.
+     */
+    url: string;
+    /**
+     * The resolved date used for sorting. Read more about [Content Dates](https://www.11ty.dev/docs/dates/).
+     */
+    date: Date;
+    /**
+     * All data for this piece of content (includes any data inherited from layouts)
+     */
+    data: {
+      authorsData: AuthorsData;
```

Comment:
I'm a bit confused, aren't these properties only on certain collections?

Reply:
This is for the items we can query from `collections` not custom made collections. Anyway, any item that is queried from `collections` has access to the `data` data...